src/ios/CDVDecimalKeyboard.m

```diff
@@ -59,7 +59,7 @@ - (void) addDecimalButton{
     decimalButton = [UIButton buttonWithType:UIButtonTypeCustom];
     [self setDecimalChar];
     [decimalButton setTitleColor:[UIColor colorWithRed:0/255.0 green:0/255.0 blue:0/255.0 alpha:1.0] forState:UIControlStateNormal];
-    decimalButton.titleLabel.font = [UIFont systemFontOfSize:40.0];
+    decimalButton.titleLabel.font = [UIFont systemFontOfSize:30.0];
```

Comment:
Also i would suggest we better allow both decimal and data-decimal, so that its compatible with old plugin.

Reply:
The font size alteration was needed in my project, I'll turn it into a preference to be in the config of the plugin, so people can use the size they want.
Also I'll alter this to allow both "data-decimal" and "decimal" to keep compatibility as you suggest.
I'll try to get around it next week as I have to check an iOS11 compatibility issue with this plugin (check the issue I wrote) and make a new pull request with your feedback added.

Thanks.